models/review.go

```diff
@@ -389,6 +413,31 @@ func GetReviewersByIssueID(issueID int64) (reviews []*Review, err error) {
 	return reviews, nil
 }
 
+// GetReviewerByIssueIDAndUserID get the latest review of reviewer for a pull request
+func GetReviewerByIssueIDAndUserID(issueID, UserID int64) (review *Review, err error) {
+	sess := x.NewSession()
```

Comment:
For select statements we should not open a new session. We use `x` directly, unless we're in the context of another session (e.g. an UPDATE operation that spans several functions), in which case we need to pass the session as a parameter.